monzo-to-qif.rb

```diff
@@ -20,7 +21,7 @@
   c.option '--settled_only', String, 'Only export settled transactions'
   c.option '--current_account', String, 'Export transactions from the current account instead of the prepaid account'
   c.action do |args, options|
-    since = options.since ? Date.parse(options.since).to_time : nil
+    since = options.since ? Date.parse(options.since).to_time : 2.weeks.ago
```

Comment:
Instead of requiring a gem for this, we can just do something like `(Time.now - (60*60*24*14)).to_date`

Reply:
Sure, so it needs changing to that in both places to prevent the empty argument causing an error.